app/config/specs/swagger2-latest-client.json

```diff
@@ -367,7 +367,9 @@
                 "consumes": [
                     "application\/json"
                 ],
-                "produces": [],
+                "produces": [
+                    "application\/json"
+                ],
```

Comment:
I think this should still be empty array

Reply:
Just set all the delete routes to use ResponseType::NONE